CockpitPlugin/build.gradle

```diff
@@ -52,6 +58,7 @@ dependencies {
 
     testImplementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
     testImplementation "org.jetbrains.kotlin:kotlin-test-junit:$kotlin_version"
+    compile files("../CockpitCore/build/libs/CockpitCore-1.1.1-SNAPSHOT.jar")
```

Comment:
what about extracting core version to some variable and use `$core_version` here?